static/gsApp/components/gsBanner.spec.tsx

```diff
@@ -2212,14 +2150,7 @@ describe('GSBanner Overage Alerts', function () {
       overagePrompts.push(`${category}_overage_alert`);
       warningPrompts.push(`${category}_warning_alert`);
       if (
-        [
-          'profile_duration',
-          'replays',
-          'spans',
-          'profile_duration_ui',
-          'seer_autofix',
-          'seer_scanner',
-        ].includes(category)
+        ['profile_duration', 'replays', 'spans', 'profile_duration_ui'].includes(category)
```

Comment:
will make sure to update product trial prompts in product trial frontend pr that will make use of `availableReservedBudgetTypes` and `canProductTrial`